### PR TITLE
Create 08_content_moderation_consent.md

### PR DESCRIPTION
**Pull request by Frijol**
A blog post from April's conversation about content moderation and consent!

For your review @dcwalk @greggish @lightandluck @lourdesvera — these posts are condensed & edited versions of our conversation, so please have a look-through and make sure I didn't put words in your mouth.

Hoping to start publishing these posts as a closeout of 2020, so please have edits in within the week if possible

### CODE REVIEW

**Comment by Frijol** [comment]
Planned publication: 12/27

File: content/posts/08_content_moderation_consent.md [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+
+**I felt responsible for this space that I'd set up,** and so I was working 18 hour days just to manage the things that all these people were doing. But a lot of the ideas were actually not very good ideas. They were all really well intentioned, but a lot of times, people just came up with something and then said, let's get to work. It was just like, Who did you talk to? Who said that was a good idea?
+
+I had a really challenging experience trying to organize this community and Slack. As fun as it was as it was coming together,** I had very few tools at my disposal for sorting through noise and for evaluating the quality **of different kinds of things that were going on. **It was only through connecting with other organizers who were experienced with this community curation and network organizing that we got some measure of control** by doing things– like, every channel should have a document that says what's happening in this channel. We had to figure out how to make it work.
```
Comment (lightandluck) [inline content/posts/08_content_moderation_consent.md]: The asterisks are off here, but not sure what was intended to be bold

File: content/posts/08_content_moderation_consent.md [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+
+Do you still feel responsible to your community?
+
+GREG: Absolutely. In the time, I felt responsible for the network that I'd convened, but also responsible to ensure that that network was accountable to members of the community. So **I was spending a lot of my time going out and finding people who would never join our Slack, talking to them, and then bringing back that perspective into this platform**.
+
+More recently, with Liz Barry, I facilitated a set of conversations discussing the phenomenon of** **emergent, spontaneous, civic hacking disaster responders. I just published a piece yesterday, [Introducing the Principles of Equitable Disaster Response](https://medium.com/@greggish/introducing-the-principles-of-equitable-disaster-response-89f0e2b44de9), that shares principles that we articulated for network-centric crisis response. It’s helpful to have something on hand to say, this is why we're going to go to the extra effort to hear from somebody before we put this out into the world.
```
Comment (lightandluck) [inline content/posts/08_content_moderation_consent.md]: Typo with asterisks

File: content/posts/08_content_moderation_consent.md [hunk 3]
```diff
@@ -0,0 +1,272 @@
...
+
+LOURDES: Has anyone heard the podcast Last Podcast on the Left? It's a comedy podcast about murders and paranormal stuff and conspiracy theories. I got really into it. I joined the the group on Facebook and started posting a lot, and all of a sudden, I was a moderator.
+
+It was a whole world; mods had our own group, and a chat. The mods purposely chose people from different perspectives, so in the chat there were these far-right dudes that were really into guns, and me, far-left lady of color; I was totally on them all the time. But we have this sense of camaraderie, too. And we even have a group name for ourselves. But it’s a couple of hours a day, and thousands of people were in this group that we modded.
+
+[The reading about the Facebook moderators](https://www.theverge.com/2019/6/19/18681845/facebook-moderator-interviews-video-trauma-ptsd-cognizant-tampa), that's a whole other level. But **there was some stuff that people posted and I saw that just made me question, what is up with humanity? And why are we, unpaid group mods, doing this?**
+
+DAWN: That was a lot of labor, Lourdes. I know.** It takes a lot of time and a lot of emotional energy.**
```
Comment (lightandluck) [inline content/posts/08_content_moderation_consent.md:98]:
```suggestion
DAWN: That was a lot of labor, Lourdes. I know. **It takes a lot of time and a lot of emotional energy.**
```

File: content/posts/08_content_moderation_consent.md [hunk 4]
```diff
@@ -0,0 +1,272 @@
...
+
+LOURDES: There's a really good book called Words that Wound, about hate speech, especially in a university setting. It's in response to this libertarian idea that all speech should be free. **It argues that words can be violent, that they have a physical impact**, and especially when it's a racist or homophobic slur or diatribe, that it can actually affect someone's livelihood.
+
+
+## Social media as a civic forum
+
+KELSEY: I'm curious how platforms play into this discussion of violence, public forums, and freedom of speech.** Facebook and Twitter feel to a lot of people like they're supposed to be a public forum**, even though they're very literally not. **Do you think that if Facebook and Twitter didn't appear to be such big civic forums, that we would request one from the government**, a digital space to discuss things?
```
Comment (lightandluck) [inline content/posts/08_content_moderation_consent.md:132]:
```suggestion
KELSEY: I'm curious how platforms play into this discussion of violence, public forums, and freedom of speech. **Facebook and Twitter feel to a lot of people like they're supposed to be a public forum**, even though they're very literally not. **Do you think that if Facebook and Twitter didn't appear to be such big civic forums, that we would request one from the government**, a digital space to discuss things?
```

File: content/posts/08_content_moderation_consent.md [hunk 5]
```diff
@@ -0,0 +1,272 @@
...
+
+DAWN: There are various governments that have attempted to create those sorts of platforms, but they tend to be a capture for consent, or resistance to ideas.
+
+Canada is a very consultation-heavy country. We get consulted a lot using consultation platforms. I think there's other countries that have models like that, like Decidim, on a different scale in Barcelona. Greece had a way that they were doing these online consultations, but not necessarily in this model of a public forum that maybe Twitter and Facebook now operate at.
+
+But in their original intent,** that’s not what Twitter or Facebook were trying to be**.
```
Comment (lightandluck) [inline content/posts/08_content_moderation_consent.md:138]:
```suggestion
But in their original intent, **that’s not what Twitter or Facebook were trying to be**.
```

File: content/posts/08_content_moderation_consent.md [hunk 6]
```diff
@@ -0,0 +1,272 @@
...
+
+**Different open source projects mean different things by “open source”, and that's okay.** It has to be okay to say that it is not the maintainer’s job to, A) make your feature, B) accept your pull request, or C) listen to you talk about what you do and don't like about the thing they gave you for free. **Open source can just mean I let you see it; open source can mean I let you suggest things but not contribute; it can mean I let you contribute; it can even mean I help you contribute. But that definition is up to the maintainer.**
+
+That understanding is frequently not reached. I think part of that has to do with open source still being kind of a new field. But part of it also has to do with people coming in with too much idealism. And I say that with a cringe in my body. But** boundaries matter a lot. If you can't make any, you don't have anything meaningful inside of them**.
```
Comment (lightandluck) [inline content/posts/08_content_moderation_consent.md:241]:
```suggestion
That understanding is frequently not reached. I think part of that has to do with open source still being kind of a new field. But part of it also has to do with people coming in with too much idealism. And I say that with a cringe in my body. But **boundaries matter a lot. If you can't make any, you don't have anything meaningful inside of them**.
```

**Review by lightandluck — APPROVED** [review]
Just some typos. Skimmed mostly, but great session!!!

**Comment by Frijol** [comment]
thanks for finding all those! Been using a google docs to markdown converter

**Pull request merged**